Formula/get_iplayer.rb

```diff
@@ -52,33 +52,6 @@ def install
 
     libexec.install "get_iplayer.cgi"
     (libexec/"get_iplayer.cgi").chmod 0444
```

Comment:
get_iplayer.cgi would need to be executable like CLI, with wrapper script like CLI. You should be able to launch it with `get_iplayer.cgi -l 127.0.0.1  -p 1935` and access in browser on <http://127.0.0.1:1935>. Presumably just need to remove chmod and replace with:

    (bin/"get_iplayer.cgi").write_env_script \
      "#{libexec}/get_iplayer.cgi",
      :PERL5LIB => "#{libexec}/lib/perl5:$PERL5LIB"